package.json

```
@@ -20,7 +20,7 @@
     "start": "yarn -s tinymce-grunt start",
     "dev": "npm-run-all oxide-icons-build oxide-build -p tsc \"tinymce-grunt dev\"",
     "ci": "npm-run-all -p oxide-ci oxide-icons-ci -p tsc \"tinymce-grunt dev\"",
-    "ci-all": "npm-run-all -p eslint ci -s tinymce-rollup",
+    "ci-all": "npm-run-all -p eslint -p strict-type-check ci -s tinymce-rollup",
```

This likely isn't what we want... we want `eslint` to run alongside `ci` not by itself. Additionally, this may fail without the oxide build happening first. I think this is what we'd want instead:
```suggestion
    "ci-all": "npm-run-all -p eslint ci -s strict-type-check tinymce-rollup",
```

---

I knew something wasn't right there